model/NewSqlResultStorage.php

```diff
@@ -97,4 +98,21 @@ public function createVariablesTable(Schema $schema)
 
         return $table;
     }
+
+    /**
+     * @inheritDoc
+     */
+    protected function getTypes(array $data = [])
```

Comment:
Are you using this method?

Reply:
Yes, please see parent class; this method used for the sake of inserts

Reply:
i think if you move this into the abstract it will make more sense and also it will remove the duplicated function required
i don't see any issue that till will cause the rds case